lldb/test/Shell/SymbolFile/PDB/add-symbols.cpp

```diff
@@ -0,0 +1,39 @@
+// REQUIRES: lld, target-windows
+
+// Test that `target symbols add <pdb>` works.
+// RUN: %build --compiler=clang-cl --nodefaultlib --output=%t.exe %s
```

Comment:
Specifically clang-cl why?

Because you want to make sure you get a PDB and it's easier to require clang-cl than it is to require some version of MSVC?